### PR TITLE
Newsweek Clone Bootstrap Project

### PR DESCRIPTION
**Pull request by JohnHernCode**
Project progress and milestones met:

    Created the skeleton for the mobile site
    Utilized Bootstrap for responsiveness
    Added content and media to the site
    Followed feedback

### CODE REVIEW

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
Hello,

I noticed that you closed the previous PR to open this particular one. You don't need to close a PR and open a new one each time you are asked to make a change. When you close a PR that has not been approved and you request a review from another PR, you make it harder for the new reviewer to follow up on the conversation on the previous PR and make it harder to review your code. Next time just push your code and submit the link to the same PR when requesting another review.

In the previous review, I listed all the changes you needed to make and so far you have only made changes to the issue of `!important` and the inline styling.
The other changes are still yet to be implemented, so please refer back to the previous review and make those changes that were required. 
Happy coding :+1: 

slack(@ebeagusamuel)

**Comment by JohnHernCode** [comment]
> 
> # Changes Required ♻️
> 
> Hello,
> 
> I noticed that you closed the previous PR to open this particular one. You don't need to close a PR and open a new one each time you are asked to make a change. When you close a PR that has not been approved and you request a review from another PR, you make it harder for the new reviewer to follow up on the conversation on the previous PR and make it harder to review your code. Next time just push your code and submit the link to the same PR when requesting another review.
> 
> In the previous review, I listed all the changes you needed to make and so far you have only made changes to the issue of `!important` and the inline styling.
> The other changes are still yet to be implemented, so please refer back to the previous review and make those changes that were required.
> Happy coding 👍
> 
> slack(@ebeagusamuel)

That was an accidental mistake. Usually that doesn't happen. Instead of commenting, it was closed. I haven't figured out GitHub that well yet.

**Comment by Mohamed-js** [comment]
@ebeagusamuel  I have changed the navbar responsiveness and made it like the original in the different screens and the body is already 0 margined and fixed the newsletter .. what lacks ?!

**Comment by Mohamed-js** [comment]
All fixed Navbar , bootstrap and linters

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1410 @@
...
+          <a class="mr-2">LOGIN</a>
+          <button class="px-2 border-0 font-weight-bold" type="submit">SUBSCRIBE ></button>
+          <button type="submit" class="navbar-toggler-icon font-weight-bold border-0 ml-2 d-inline-block mr-0 p-3"></button>
+        </form>
+      </nav>
+
+      <nav class="navbar nav2 container navbar-dark font-weight-bold my-0 py-0 mx-auto align-text-bottom">
+        <form class="form-inline">
+          <div>
+            <a href=""><i class="fas fa-cloud"></i> Alexandria, EG</a>
```
Comment (umairarshadbutt) [inline index.html]: Add the description in all `href` attribute of index page

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1410 @@
...
+            <p>Tue, Dec 08, 2020</p>
+          </div>
+        </form>
+        <a style="max-width: 90%; margin: 0;" class="navbar-brand" href="#"><img style="max-width: 100%; margin: 0;" src="imgs/main/logonews.png" alt=""></a>
```
Comment (umairarshadbutt) [inline index.html]: Add the description in all `alt` attribute of the index page

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1410 @@
...
+          <li><a href="">Tech & Science</a></li>
+          <li><a href="">Culture</a></li>
+          <li><a href="">Newsgeek</a></li>
+          <li><a href="">Sports</a></li>
+          <li><a href="">Health</a></li>
+          <li><a href="">The Depate</a></li>
+          <li><a href="">vantage</a></li>
+          <li><a href="">weather</a></li>
+          <li>
+            <input class="border-0" style="max-width: 50px;" type="text" placeholder="Search">
```
Comment (umairarshadbutt) [inline index.html]: Remove inline CSS

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Mohamed-js  and @JohnHernandez-code 
Such a remarkable project you made :+1: ! Awesome README :100: !
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [x] Remove inline CSS from index.html file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Add the description in all `alt` attribute of the index page (See the inline comment)
- [x] Add the description in all `href` attribute of the index page (See the inline comment)
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: index.html (hunk 1 above)
Comment (JohnHernCode) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (JohnHernCode) [reply index.html]: Done

**Comment by Mohamed-js** [comment]
@umairarshadbutt  all is done ✅

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @JohnHernandez-code and @Mohamed-js  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] You are missing `THE DEBATE` section 


| Your clone | Original page |
| --- | --- |
| :grey_question: :grey_question: :grey_question:  | Screenshot from 2020-12-09 20-50-20 |

- [x] The login link should change into user icon on mobile screens 

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 23-06-31| Screenshot from 2020-12-09 20-47-27 |

:point_right: Could you see the difference :question: :question: :question: 

- [x] The picture which is on the leftmost side of the subscribe section, should be two overlapping pictures


| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-09 23-04-19| Screenshot from 2020-12-09 23-05-14|

:point_right: Could you see the difference :question: :question: :question: 

# [Optional changes] :yellow_circle:

:warning: Your commit messages should be in imperative mode or present tense, _i.e.:_ you should have written `Fix linters and bootstrap` instead of writing `Fixed linters and bootstrap`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by Mohamed-js** [comment]
All requested changes done, thank you.

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hello team @JohnHernandez-code and @Mohamed-js 👋 👋 , great job on working on the previous TSE comments 👏 👏 
Don't get discouraged by all the changes you are being asked to do, you are almost there, now give me a smile😄 

Check out the comments below ⬇️ 

- [ ] In the mobile view, the article order should be Top story, featured story, debate section, opinion section, more stories, and lastly my turn section.  Please re-order your articles in the correct order 😄 [link](http://newsweek.com/)

- [ ] in the mobile view, the second article should be placed right below the first article. Check out the screenshot below:

Original

image

Clone
image

Notice the difference? Also, the text should be aligned below the images. Kindly check on that

- [ ] On the tablet view, hide the user icon. [link](http://newsweek.com/)

Clone:

image

That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to reach out on slack if you have any questions

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @Mohamed-js & @JohnHernandez-code**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] right now your footer looks like this:
asdasdasd

you can fix this issue by adding a container class to your footer tag.


- [ ] both of these hr's have a bottom margin of 1rem. please make sure to add it to the bottom one only.
sadasdsadgagahsaash


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @JohnHernandez-code  & @Mohamed-js**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**